platforms/jvm/normalization-java/src/main/java/org/gradle/internal/normalization/java/impl/package-info.java

```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright 2020 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-@NonNullApi
-package org.gradle.internal.normalization.java.impl;
```

Comment:
💭 These were removed because `@NonNullApi` is not available in `:build-logic`. I don't think we should move it there either. Instead, we should find an existing annotations library that includes something like our `@NonNullApi` and use that. Or better yet, make it so our default behavior is non-null, and then nothing needs to be annotated.